### ISSUE TITLE
Show total balance from API in account selector

### ISSUE THREAD

**Issue by lukaw3d** [oasisprotocol/oasis-wallet-web#1110]
Currently we only show available balance from gRPC

https://github.com/oasisprotocol/oasis-wallet-web/blob/9719ee006b0fd95266a7f5c47329d3b3beb391c2/src/app/pages/OpenWalletPage/Features/ImportAccountsSelectionModal/index.tsx#L32

https://github.com/oasisprotocol/oasis-wallet-web/blob/550f447ff6ddaa213bff80c57db01f6573df5852/src/app/components/Toolbar/Features/AccountSelector/index.tsx#L106

**Comment by csillag** [comment]
Unfortunately the problem is not merely not showing all the data; we don't even load the required data properly.
For example, look at this account:

http://localhost:3000/account/oasis1qpzregxdxcu83yj4zzvadrhysntunpe44u66s4e5

The balance is:

![image](https://user-images.githubusercontent.com/2093792/201243053-13aa2755-c69d-4a6a-8c34-5af01b9bc334.png)

However, when we load the balance for this account, we are running:

``` TypeScript
  const account = yield* call([nic, nic.stakingAccount], {
    height: 0,
    owner: short,
  })
  console.log('Loaded account', JSON.stringify(account, null, '  '))
  const balance = parseRpcBalance(account)
  console.log('loaded balance', JSON.stringify(balance, null, '  '))
  return balance

```

And this gets us:

```
Loaded account {
  "general": {
    "nonce": 1,
    "balance": {
      "0": 2,
      "1": 84,
      "2": 11,
      "3": 228,
      "4": 0
    }
  }
}
loaded balance {
  "available": "10000000000",
  "validator": {
    "escrow": "0",
    "escrow_debonding": "0"
  }
}
```

... so as we can see, the staked part of the balance is missing.

(The same thing happens with both `OasisScan` and `OasisMonitor`.)

**Comment by lukaw3d** [comment]
gRPC doesn't have the needed information, but APIs do

https://api.oasisscan.com/testnet/chain/account/info/oasis1qpzregxdxcu83yj4zzvadrhysntunpe44u66s4e5
https://monitor.oasis.dev/api/testnet/data/accounts/oasis1qpzregxdxcu83yj4zzvadrhysntunpe44u66s4e5

A while ago I added parsing for this from API and measured how outdated the info is: https://github.com/oasisprotocol/oasis-wallet-web/commit/f1a2d867c93db0375f3167d8c08703d9670f853a
That should come in useful
